defold-spine/editor/src/spineext.clj

```diff
@@ -346,9 +346,23 @@
 (defn- set-constants! [^GL2 gl shader ro]
   (doall (map (fn [constant] (set-constant! gl shader constant)) (.m_Constants ro))))
 
+(defn- blend-factor-value-to-blend-mode [blend-factor-value]
+       (case blend-factor-value
+             0 :blend-mode-alpha
+             1 :blend-mode-add
+             2 :blend-mode-mult
+             3 :blend-mode-screen
+             :blend-mode-alpha))
+
```

Comment:
Style: Indentation got messed up because you didn't have the Spine Extension source directory in `:source-paths` when you wrote this. Now that you have, **Code > Auto-Indent Lines** should sort it out!